cli/crates/server/src/errors.rs

```diff
@@ -87,6 +87,10 @@ pub enum ServerError {
     #[error("could not parse grafbase/schema.graphql\n{0}")]
     ParseSchema(String),
 
+    /// returned if the typescript config parser command exits unsuccessfully
+    #[error("could not load grafbase/grafbase.config.ts\n{0}")]
```

Comment:
on newer errors we typically do something like `caused by: {0}`, I'd copy one of those

Reply:
Although it'd depend if the error output here makes sense in that context

Reply:
So its clear that the errors are related